test/programExecutionEngineTest.cpp

```diff
@@ -428,9 +372,8 @@ TEST_F(ProgramExecutionEngineTest, execute)
     Program::ProgramExecutionEngine progExecEng(*p);
     double result;
 
-    double r6 = (value0 + value1 + value0 + value0) / 4;
-    double r1 = value0 + r6;
-    double r0 = r1 * (Parameter(value1)).operator float();
+    double r1 = value0;
+    double r0 = r1;
     r0 = r0 * value2 + r1 * value3;
```

Comment:
Why changing the program so much.. 
Was this absolutely necessary.

Reply:
It has been required during the development as the support on the parameters and the MultByConst instruction were temporarily removed. It has accidentally been kept this way afterwards

Reply:
The program may have been complex for a reason (I'm not sure though). Don't hesitate to add a bit of complexity again :)